string topic
string camera_frame_id
string client_id
---
string topic
